### PR TITLE
[FEM] add new example file for flow and heat

### PR DESCRIPTION
**Pull request by donovaly**
- also extend examples libraries for (more to come) examples with multiphysics (multiple equations)

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8550">pipeline status</a> for feature branch [PR_8550](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8550). Pipeline [782641976](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/782641976) was triggered at [41af6b9](https://github.com/FreeCAD/FreeCAD/pull/8550/commits/41af6b95889056af1eb7cf0a67e84034cfb75889). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8550">pipeline status</a> for feature branch [PR_8550](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8550). Pipeline [782642800](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/782642800) was triggered at [32c8c42](https://github.com/FreeCAD/FreeCAD/pull/8550/commits/32c8c42642fdf67a31592b9e70458dc911088c02). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Fem/femexamples/equation_flow_elmer_2D.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    circle = Draft.make_circle(10, placement=placementCircle)
+    circle.Label = "HeatingRod"
+    circle.ViewObject.Visibility = False
+
+    # a link of the circle
+    circleLink = doc.addObject("App::Link", "Link-HeatingRod")
+    circleLink.LinkTransform = True
+    circleLink.LinkedObject = circle
+
+    # cut rod from wire to get volue of fluid
```
Comment (luzpaz) [inline src/Mod/Fem/femexamples/equation_flow_elmer_2D.py:96]: `volume`

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8550">pipeline status</a> for feature branch [PR_8550](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8550). Pipeline [782649208](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/782649208) was triggered at [c1d694b](https://github.com/FreeCAD/FreeCAD/pull/8550/commits/c1d694b9f25a854156d2e0de3e8fc5f7424a2f41). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Fem/femexamples/equation_flow_elmer_2D.py (hunk 1 above)
Comment (donovaly) [reply src/Mod/Fem/femexamples/equation_flow_elmer_2D.py]: thanks fixed

**Pull request merged**